### PR TITLE
Upload Changes

### PR DESCRIPTION
**Pull request by fitigh91**

### CODE REVIEW

**Comment by dachafra** [comment]
Assign3 correct

File: Assignment2/fitigh91.ttl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ 
+:class01 vocabClass:includes :sensor029;
+		 vocabClass:includes :computer101.
+		 
+:sensor029 vocabSensor:hasMeasurement :measurement8401.
+
+:computer101 vocabPc:hasOwner :user10A.
+
+:measurement8401 vocabMess:hasTemperature :"29^^xsd:integer";
+				 vocabMess:atTime: "2010-06-12T 12:00:12^^xsd:dateTime".
```
Comment (ocorcho) [inline Assignment2/fitigh91.ttl]: This is wrong.

File: Assignment2/fitigh91.ttl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@prefix vocabOwner: <http://www.ontologies.org/ontologies/class01/sensor029/user10A#>
+ 
+:class01 vocabClass:includes :sensor029;
+		 vocabClass:includes :computer101.
+		 
+:sensor029 vocabSensor:hasMeasurement :measurement8401.
+
+:computer101 vocabPc:hasOwner :user10A.
+
+:measurement8401 vocabMess:hasTemperature :"29^^xsd:integer";
```
Comment (ocorcho) [inline Assignment2/fitigh91.ttl]: The prefix for hasTemperature does not make sense.

**Comment by ocorcho** [comment]
Check comments

**Pull request closed without merging**